AwsEncryptionSDK/runtimes/go/examples/readme.md

```diff
@@ -0,0 +1,90 @@
+# AWS Encryption SDK for Go Examples
+
+This section features examples that show you
+how to use the AWS Encryption SDK.
+We demonstrate how to use the encryption and decryption APIs
+and how to set up some common configuration patterns.
+
+## APIs
+
+The AWS Encryption SDK provides two high-level APIs:
+one-step APIs that process the entire operation in memory
+and streaming APIs.
+
+You can find examples that demonstrate these APIs
+in the [`examples/`](./) directory.
+
+* [How to encrypt and decrypt](./keyring/awskmskeyring/awskmskeyring.go)
+* [How to change the algorithm suite](./misc/setencryptionalgorithmsuite.go)
+* [How to set the commitment policy](./misc/commitmentpolicy.go)
+* [How to limit the number of encrypted data keys (EDKs)](./misc/limitencrypteddatakeysexample.go)
+
+## Configuration
+
+To use the encryption and decryption APIs,
+you need to describe how you want the library to protect your data keys.
+You can do this by configuring
+[keyrings](#keyrings) or [cryptographic materials managers](#cryptographic-materials-managers).
+These examples will show you how to use the configuration tools that we include for you
+and how to create some of your own.
+We start with AWS KMS examples, then show how to use other wrapping keys.
+
+* Using AWS Key Management Service (AWS KMS)
+    * [How to use one AWS KMS key](./keyring/awskmskeyring/awskmskeyring.go)
+    * [How to use multiple AWS KMS keys in different regions](./keyring/awskmsmrkmultikeyring/awskmsmrkmultikeyring.go)
+    * [How to decrypt when you don't know the AWS KMS key](./keyring/awskmsdiscoverykeyring/awskmsdiscoverykeyring.go)
+    * [How to limit decryption to a single region](./keyring/awskmsmrkdiscoverykeyring/awskmsmrkdiscoverykeyring.go)
+    * [How to decrypt with a preferred region but failover to others](./keyring/awskmsmrkdiscoverykeyring/awskmsmrkdiscoverykeyring.go)
+    * [How to reproduce the behavior of an AWS KMS master key provider](./keyring/awskmsmultikeyring/awskmsmultikeyring.go)
+* Using raw wrapping keys
+    * [How to use a raw AES wrapping key](./keyring/rawaeskeyring/rawaeskeyring.go)
+    * [How to use a raw RSA wrapping key](./keyring/rawrsakeyring/rawrasakeyring.go)
+* Combining wrapping keys
+    * [How to combine AWS KMS with an offline escrow key](./keyring/multikeyring/multikeyring.go)
+* How to restrict algorithm suites
+    * [with a custom cryptographic materials manager](./cryptographicmaterialsmanager/restrictalgorithmsuite/signingsuiteonlycmm.go)
+
+### Keyrings
```

Comment:
Could you link to https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html?